src/navsat_transform_node.cpp

```diff
@@ -37,11 +37,14 @@ int main(int argc, char ** argv)
 {
   rclcpp::init(argc, argv);
 
-  auto node = rclcpp::Node::make_shared("navsat_transform_node");
-
-  robot_localization::NavSatTransform trans(node);
-
-  trans.run();
+  rclcpp::executors::SingleThreadedExecutor exec;
+  
+  const rclcpp::NodeOptions options;
+  auto navsat_transform_node = std::make_shared<robot_localization::NavSatTransform>(options);
+  
+  exec.add_node(navsat_transform_node);
```

Comment:
if its just 1 node to spin you can `rclcpp::spin(navsat_transform_node->get_node_base_interface());` without the executor unless you want multi-threaded